### PR TITLE
Blog post

### PR DESCRIPTION
**Pull request by mcullan**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,4 +8,3 @@ _site
 .tweet-cache
 Gemfile.lock
 vendor
-
```
Comment (busycalibrating) [inline .gitignore:11]: Can you revert these changes?

File: _posts/2023-02-10-hippo-to-h3.md [hunk 2]
```diff
@@ -0,0 +1,772 @@
...
+0 & n \leq k
+\end{cases}
+$$
+
+$$
+B_n = 1
+$$
+
+### HiPPO-LegS: Legendre Scale-invariant
+{% include figure.html path="assets/img/2023-02-10-hippo-to-h3/legs.png" class="img-fluid" %}
```
Comment (busycalibrating) [inline _posts/2023-02-10-hippo-to-h3.md:233]: I think this needs to be `... path="assets/img/2023-02-10-hippo-to-h3/legs.png ...`

**Review by busycalibrating — COMMENTED** [review]
Thanks for your submission! There just 2 minor things to fix and then I'll merge it in!

File: .gitignore (hunk 1 above)
Comment (mcullan) [reply .gitignore]: Reverted! Not sure how that happened there. Forgot I could just git rebase those commits away

File: _posts/2023-02-10-hippo-to-h3.md (hunk 2 above)
Comment (mcullan) [reply _posts/2023-02-10-hippo-to-h3.md]: I'm not sure if I follow... those look like the same path to me. 

But I noticed the .png in assets capitalized the s in "legS" in the filename. It didn't seem to cause any issues on the post, as far as case-sensitivity, but I updated it to match.

Comment (busycalibrating) [reply _posts/2023-02-10-hippo-to-h3.md]: Sorry that's what I meant/thought I wrote. If you're using a Mac then I think capitalization doesn't matter, but it does on linux. In any case thanks I think this should be good!

**Comment by busycalibrating** [comment]
Great I think everything is good now, I'll merge it in. Thanks for the submission!
